packages/react-variant-view/src/component/cancerPatientPopulation/CancerPatientPrevalence.tsx

```diff
@@ -0,0 +1,195 @@
+                                    <DefaultTooltip
+                                        placement="top"
+                                        overlay={
+                                            <span>
+                                                {
+                                                    PathogenicityNameHelper[
+                                                        Pathogenicity.BIALLELIC
+                                                    ].description
+                                                }
+                                            </span>
+                                        }
+                                    >
+                                        <span
+                                            className={
+                                                featureTableStyle[
+                                                    'data-source-without-linkout'
+                                                ]
+                                            }
+                                        >
+                                            {
+                                                PathogenicityNameHelper[
+                                                    Pathogenicity.BIALLELIC
+                                                ].title
+                                            }
+                                        </span>
+                                    </DefaultTooltip>
+                                    {`: `}
+                                    {this.fixDisplayValue(
+                                        mutation.ratioBiallelicPathogenic
+                                    )}
+                                    {`%)`}
+                                </span>
+                            )
+                        );
+                    }
+                } else if (isSomaticMutation(mutation)) {
+                    contentMap.set(
+                        Pathogenicity.SOMATIC,
+                        this.percentageContent(
+                            Pathogenicity.SOMATIC,
+                            this.fixDisplayValue(mutation.somaticFrequency)
+                        )
+                    );
+                }
+            });
+
+            // show germline on the top
+            if (contentMap.get(Pathogenicity.GERMLINE)) {
+                content.push(contentMap.get(Pathogenicity.GERMLINE)!);
+            }
+            if (contentMap.get(Pathogenicity.SOMATIC)) {
+                content.push(contentMap.get(Pathogenicity.SOMATIC)!);
+            }
+        }
+        if (content.length === 0) {
+            content.push(
+                <div key={'cancer-patient-prevalence-no-data'}>
+                    No data available
+                </div>
+            );
+        }
+        return content;
+    }
+
+    // duplicate of FrequencyCell.tsx mainContent()
+    private fixDisplayValue(value: number | null) {
```

Comment:
We should move this to a utility/helper file.

Reply:
I move this one to `cbioportal-frontend-commons` too, because the `FrequencyCell.tsx` there also needs this function

Reply:
Moved to `cbioportal-utils` in the end, this is a util function so might be more proper there

Reply:
Makes sense. We should probably find a more descriptive name for the function. Its main purpose is to convert values like `0.004` to `< 0.01`. I don't know which name describes this conversion best though.

Also, we should add a second parameter `fractionDigits: number = 1` to make the function more generic.